kalamine/template.py

```diff
@@ -271,6 +270,7 @@ def klc_virtual_key(layout: "KeyboardLayout", symbols: list, scan_code: str) ->
         # We affect abitrary OEM VK and it will not match the one
         # in distributed layout. It can cause issue if a application
         # is awaiting a particular OEM_ for a hotkey
+        global oem_idx
         oem_idx += 1
```

Comment:
Ugh, I missed it in my previous review. A decent linter would’ve caught it, I’ll try to add that to the CI.